segment-event-notification/.env

```diff
@@ -0,0 +1,15 @@
+# description: The name of the Segment event you want to be notified about
+# format: text
+# link: https://segment.com/docs/connections/spec/track/#event
+# required: true
+SEGMENT_EVENT=
+
+# description: The Twilio phone number to send SMS notifications from
+# format: phone_number
+# required: true
+TWILIO_PHONE_NUMBER=+12223334444
+
+# description: The number you want to send notifications to
```

Comment:
we recently landed some changes that improved the language we use around users entering their phone number that it would be good to keep consistent https://github.com/twilio-labs/function-templates/blob/main/forward-message/.env#L1-L5

Reply:
Updated to the language from that `.env` file.